app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -307,7 +307,12 @@ export const ListingDetailsEligibility = ({
         {isRental(listing) && (
           <ListSection
             title={t("listingsForRent.rentalAssistance.title")}
-            subtitle={t("listingsForRent.rentalAssitance.subtitle")}
+            subtitle={
+              <>
+                <div className="mb-4">{t("listingsForRent.rentalAssistance.info1")}</div>
+                {t("listingsForRent.rentalAssistance.info2")}
```

Comment:
For consistency, please wrap this string in a `div` as well.

No need to address in this PR, but it doesn't feel right to have `div` elements nested inside the inline `span` element, maybe an accessibility audit will flag this in the future? For some reason the "Household Maximum Income" section never got flagged during an audit, it has several `divs`, an `ul`, and `li`s all nested in a `span` 😲

Reply:
Tangential to this PR, but putting it down before I forget:

This bloom component could do with some changes: https://github.com/bloom-housing/ui-components/blob/main/src/sections/ListSection.tsx

1. `<span>` might be better off replaced with `<p>`
2. to avoid nesting non-string subtitles, do something like this:
    ```js
    <hgroup>
      <h3 className="custom-counter__title">{props.title}</h3>
      {typeof props.subtitle == 'string' ? (
        <span className="custom-counter__subtitle">{props.subtitle}</span>
      ) : (
        <>
          props.subtitle
        </>
      )}
    </hgroup>
    ```

Reply:
@cliu02 Yes, I mean the translation string, like this
```js
  <>
    <div className="mb-4">{t("listingsForRent.rentalAssistance.info1")}</div>
    <div>{t("listingsForRent.rentalAssistance.info2")}</div>
```